platform/working-with-vsp/roles-and-responsibilities/accessibility.md

```diff
@@ -1,29 +1,87 @@
+- Maintains accessibility compliance in common code and libraries
+- Conducts accessibility reviews for applications and static content to identify both launch-blocking issues ([Severity 0](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-0) and [Severity 1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)) as well as other [post-launch improvements](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#post-launch-issues) (deeper dive with more devices, tools)
+- Provides support for accessibility questions via Slack and accessibility office hours
+
+## VFS team responsibilities
+
+- Ensure their applications and code meet [VA.gov accessibility standards](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility#depo-platform-accessibility-standards)
+- Work with the Veteran-facing Services Platform (VSP) team using the [Collaboration Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/working-with-vsp/vsp-collaboration-cycle) to ensure accessibility standard compliance for:
+  - New UI components
+  - Changes to existing UI components
+  - User flow patterns
+  - Larger content changes
+- Run [foundational accessibility tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) on their applications, including:
+  - Provide documentation in [TestRail](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc) or [Github](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/accessibility-test-cases-template.md) certifying tests were run
+  - Fix accessibility issues in application code and large changes to static content, including content generated by Drupal or Markdown.
+- Reduce the backlog of post-launch accessibility improvements over time, as prioritized by their DEPO product lead
+
+## Collaboration cycle for VFS teams with an accessibility specialist
+
+These are the expected responsibilities and division of labor for teams with one or more accessibility specialists. This section will outline each collaboration cycle meeting.
+
+- **[Project Kickoff](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md#project-kickoff)**
+
+  - VFS accessibility specialists are encouraged to attend the project kickoff meeting. Project kickoffs are a good opportunity for teams to ask questions and learn about the new product or feature.
+  - DEPO platform will collect specialist feedback (if any) and provide it to the VFS product team
```

Comment:
What do you think about saying "...collect and review specialist feedback..."?

Reply:
👍